server/homebrew.api.js

```diff
@@ -62,6 +57,14 @@ const getBrew = (accessType, stubOnly = false)=>{
 			// Combine the Homebrewery stub with the google brew, or if the stub doesn't exist just use the google brew
 			stub = stub ? _.assign({ ...excludeStubProps(stub), stubbed: true }, excludeGoogleProps(googleBrew)) : googleBrew;
 		}
+		const authorsExist = stub?.authors?.length > 0;
+		const isAuthor = (stub?.authors || [])?.includes(req.account?.username);
```

Comment:
Do we need the `|| []` ? If its an empty array it can't include the current username.
```
const isAuthor = stub?.authors?.includes(req.account?.username);

or

const isAuthor = authorsExist && stub.authors.includes(req.account?.username);
```

Reply:
Yeah for some reason I thought that js would barf at the method call on undefined but I just double checked and it seems fine

Reply:
I think the Elvis stops evaluating as soon as it sees a property doesn't exist so it doesn't even get far enough to call the method.